driver/pace/driver/run.py

```diff
@@ -508,10 +586,22 @@ def step_all(self):
             self._step(timestep=self.config.timestep.total_seconds())
             time += self.config.timestep
             self.diagnostics.store(time=time, state=self.state)
+        self.performance_config.collect_performance(
+            self.performance_config.timestep_timer,
+            self.performance_config.times_per_step,
+            self.performance_config.hits_per_step,
+        )
```

Comment:
If you'd like to avoid passing these as arguments twice, you could use the attributes on `self` in a `performance_config.collect_performance()`.